TEAMSchools/teamster
# kippnewark

- dockerignore
- cp miami
- update resources
- rename to local
- fix env
- fix local
- fix configs
- ps query
- add now
- more queries
- breakup query
- fix bytes
- add fixed queries back
- fix configs
- update config
- rework configs
- try new gcs fm
- use stock gcs fm
- configs
- remove
- fmt
- merge dev
- del configs
- resync job
- add extract configs
- fmt
- rm
- write sql to files
- add log
- del data to save memory
- del pt to conserve mem
- query kwargs
- fix json encoding
- logs
- make file persistent
- decrease partition size
- skip output if gcs
- change partition size
- ts name fmt
- fix config
- fix config
- update deps, ssh prot env
- refactor transform
- drop transform
- edit config
- remove full tables from reync
- connect args
- retrypolicy
- timelimit
- rm connect args
- fix max_concurr
- breakup years, up retries
- fix query_kwargs
- fix ts str
- fix config
- fix gs names
- change today
- fix imports
- merge dev
- cleanup
- remove k8s tags
- rm docker
- fix config
- test query
- broken ps jobs
- update repo
- run multigraph
- declaritve graphs
- fix config map
- fix config
- fix config
- fix file output fmt
- refactor ssh
- fix config map
- rm merge dict
- map partitions
- multiprocess partitions
- undo multiproc
- multiproc partitions
- fix len
- reduce pool size
- rm multiprocess
- refactor partition size, update config
- fix gcs folder
- fix output fmt
- fix fm key
- fix graph name
- fix fm key
- add blob check
- refactor where clause fmt
- trblsht
- fix fm key
- last run today
- rm . from suffix
- use correct method
- fix blob exists
- debug
- debug
- list_blob params
- rm prefix
- finally figure out blob list
- rm extra /
- add retry policy
- add retry policy
- fix where fmt
- fix config
- fill out config
- dockerfile hotfix
- disable_oob
- rm connect kwargs
- break up assignmentscore
- fix config
- resync vs standard
- rename
- skip resync during subsequent runs
- refactor where clause fmt
- breakout pgf and att, remove unnecessary config
- fix config map
- fix blob lookup
- rm partition size from config
- job name
- rearrange
- trunk
- refactor resync
- use stock gcs
- fix resync partitions
- ext jobs
- config
- refactor ext runs
- refactor
- docker
- rename
- fix all
- resources
- resources
- :lipstick:
- refactor datagun
- file config req
- combine tests, fix whereclause
- fix graph
- fix sep, stem
- refactor local dataguns
- default path
- fix naming
- merge
- ps sched
- config
- microsecs
- fmt
- refactor graph creation, resync to local
- refactor count
- fix output type
- optional out
- config map count tables
- fix sync config
- fix config
- fix config map
- trblsht counts
- refactors
- count
- fix config map
- skip count when no where clause
- refactor
- op factory
- graph factory
- fix config map
- extensions
- fix resync
- alias get_counts
- fix alias
- fix aliases
- whoops
- fix config
- fix config map
- fix config
- fix schem,a
- config
- fix sql composition logic
- add ext config
- fix path
- fix config
- fix dict merge
- SO link
- fix pythonpath
- fix resync output name
- rm
- trunk
